config.go

```diff
@@ -183,6 +183,10 @@ type Config struct {
 			// The initial offset to use if no offset was previously committed.
 			// Should be OffsetNewest or OffsetOldest. Defaults to OffsetNewest.
 			Initial int64
+
+			// The retention duration for committed offsets.
+			// A value of 0 cause the retention time to default to the offsets.retention.minutes broker option.
```

Comment:
Please follow the style of other option fields:
- keep lines short enough to display properly on godoc.org
- explicitly list the default value
- mention that Kafka only supports precision up to milliseconds; nanoseconds will be truncated.
- add a warning to the `Validate` method if greater-than-millisecond precision is provided